src/component/card/dataverseCard/dataverseCard.tsx

```diff
@@ -27,6 +28,8 @@ export const DataverseCard: FC<DataverseCardProps> = ({ type, label, description
         return 'primary-color-variant-3'
       case 'dataset':
         return 'primary-color-variant-4'
+      default:
```

Comment:
As the type of `DataverseItem` is one of `'dataspace' | 'dataset' | 'service'` we don't need a default value here because every case is handled.